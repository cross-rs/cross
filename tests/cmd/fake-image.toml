bin.name = "cross"
args = ["build"]
status = "failed"

[env.add]
CROSS_TARGET_X86_64_UNKNOWN_LINUX_GNU_IMAGE = "ghcr.io/cross-rs/fake-image:main"
