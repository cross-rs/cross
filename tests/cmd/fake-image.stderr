[..]ghcr.io/cross-rs/fake-image:main[..]
...